src/xregexp.js

```diff
@@ -1157,6 +1157,8 @@ XRegExp.matchChain = function(str, chain) {
  *       backreference n/nn.
  *     - ${n} - Where n is a name or any number of digits that reference an existent capturing
  *       group, inserts backreference n.
+ *     - $<n> - Where n is a name or any number of digits that reference an existent capturing
```

Comment:
See above where `$&, $0` are on the same line. Would you mind doing the same here, and grouping this with `${n}`?